CONTRIBUTING.md

```diff
@@ -1,35 +1,36 @@
 How to report a bug
 -------------------
 - Use the search function to see if someone else has already reported the bug. If an issue has already been opened, see if you can add any useful information to it.
-- Otherwise, create a new issue on the "issues" page
-- Describe the problem as detailed as possible.
-- Some bugs only occur on certain devices or versions of Android. Please add information about your device and the version of Android that is running on it (you can look these up under `Settings → About Phone`) as well as which version of AntennaPod you are using.
-- If the bug only seems to occur with a certain podcast, make sure to include the URL of that podcast.
+- Otherwise, create a new issue on the "issues" page.
+- Describe the problem with as much detail as possible.
+- Some bugs may only occur on certain devices or versions of Android. Please add information about your device and the version of Android that is running on it (you can look these up under `Settings → About Phone`), as well as which version of AntennaPod you are using.
+- If the bug only seems to occur with a certain podcast, please include the URL of that podcast.
 - If possible, add instructions on how to reproduce the bug.
-- If possible, add a logfile to your post. This is especially useful if the bug makes the application crash. You can create logfiles with an app like `aLogcat`. Just launch `alogcat`, then start AntennaPod and reproduce the bug. aLogcat should now display information about the bug when you start it.
-- Usually, you can make a screenshot of your smartphone by pressing [Power] + [Volume down] for a few seconds
+- If possible, add a logfile to your post. This is especially useful if the bug makes the application crash. You can create logfiles with apps like `aLogcat` (or [Scoop](https://forum.xda-developers.com/android/apps-games/adb-root-scoop-catch-stack-trace-app-t3888798)). Just launch `alogcat`, then start AntennaPod and reproduce the bug. aLogcat should now display information about the bug when you start it.
+- Usually, you can take a screenshot of your smartphone by pressing *Power* + *Volume down* for a few seconds.
 - Please use the following **template**: 
```

Comment:
I think this copy of the template can be removed and be replaced with a link to the files in the `.github` folder to prevent inconsistencies

Reply:
Gotcha.